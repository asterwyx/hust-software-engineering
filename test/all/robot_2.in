preset user AQS

preset credit A 10
preset credit Q 10
preset credit S 10

preset gift A barrier 1
preset gift Q robot 1
preset gift S robot 1

preset barrier 55

preset userloc A 30 0
preset userloc Q 15 0
preset userloc S 32 0

preset nextuser S

robot #S
step 10 #S:32->42
n
block -3 #barrier at 27
step 7 #A:30->37
n
robot # do not remove barrier at 27
step 22 #Q:15->37 through barrier at 27
n

dump 
