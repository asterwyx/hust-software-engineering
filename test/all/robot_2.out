user AQS

fund A 10000
fund Q 10000
fund S 10000

credit A 10
credit Q 10
credit S 10

barrier 55

userloc A 37 0
userloc Q 27 0
userloc S 42 0

nextuser S

